### PR TITLE
ctr-remote: add ctr-remote command

### PR DESCRIPTION
**Pull request by luodw**
There are some cases user use ctr to pull image and run container,
so here add ctr-remote command to pull nydus image refer to stargz
https://github.com/containerd/stargz-snapshotter/tree/main/cmd/ctr-remote

Signed-off-by: luodaowen.backend <luodaowen.backend@bytedance.com>

### CODE REVIEW

**Review by bergwolf — APPROVED** [review]
Thanks a lot!

File: contrib/ctr-remote/commands/rpull.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
+/*
+   Copyright The containerd Authors.
+
```
Comment (changweige) [inline contrib/ctr-remote/commands/rpull.go:3]: Is keeping license header here suitable?

Comment (jiangliu) [reply contrib/ctr-remote/commands/rpull.go:3]: I feel it's OK:)

Comment (luodw) [reply contrib/ctr-remote/commands/rpull.go:3]: As the code is refer to stargz mostly，so keep the license header here

**Pull request merged**